.travis.yml

```diff
@@ -85,19 +83,18 @@ jobs:
       compiler: gcc
       env: ARCH=x86_64
       before_install:
-        - "wget 'https://raw.githubusercontent.com/alpinelinux/alpine-chroot-install/v0.7.0/alpine-chroot-install' \
-              && echo '090d323d887ef3a2fd4e752428553f22a52b87bb  alpine-chroot-install' | sha1sum -c || travis_terminate 1"
-        - alpine() { /alpine/enter-chroot -u "$USER" "$@"; }
+        - docker run -w /src -dit --name alpine -v $(pwd):/src node:10-alpine
+        - alpine() { docker exec -it alpine "$@"; }
```

Comment:
i know almost nothing about docker as well as alpine :) but what do these changes do?

Reply:
Ah this is switching the alpine chroot to basically using a docker image instead. It was easier at least for me to find a Node.js image using Alpine than it was to figure out the chroot script. Functionally they're basically the same though with this `alpine` shim function